api/v1alpha1/types.go

```diff
@@ -74,6 +74,10 @@ type Argument struct {
 type CombinationStatus struct {
 	// Conditions represents the current condition of the Combination.
 	Conditions []metav1.Condition `json:"conditions,omitempty"`
+	// Represents the evaluation to this combination once processed
+	Evaluations []string `json:"evaluations,omitempty"`
+	// Phase represents a human-readable description of where the combinations is
+	Phase string `json:"phase,omitempty"`
```

Comment:
I think this is worth pursuing but I'm not entirely convinced we need it yet, or need to implement (and document) the various transistions in a state machine. Any thoughts?

Reply:
I'm good with removing it and making it a follow-up. I would also advocate that it definitely helps the UX.

Reply:
Removed